geps/overview.md

```diff
@@ -61,18 +61,45 @@ against them. At this stage, the GEP should be targeting the "Implementable"
 stage.
 
 ### 4. Implement the GEP as "Experimental"
+
 With the GEP marked as "Implementable", it is time to actually make those
 proposed changes in our API. In some cases, these changes will be documentation
 only, but in most cases, some API changes will also be required. It is important
 that every new feature of the API is marked as "Experimental" when it is
 introduced. Within the API, we use `<gateway:experimental>` tags to denote
-experimental fields.
+experimental fields. In order to transition a GEP into `Experimental` **the
+graduation criteria for the GEP MUST be filled out**, indicating all of the
+currently known issues and dilemmas that need to be resolved in order to
+graduate to `Standard`.
 
 Before these changes are released, they will also need to be documented.
 GEPs that have not been both implemented and documented before a release
 cut off will be excluded from the release.
 
-### 5. Graduate the GEP to "Standard"
+### 5. Probationary Period
```

Comment:
It may help to provide a sentence or paragraph that describes the rationale for this - essentially trying to avoid the state of features getting stuck in experimental forever.

Reply:
Added in 2cc4acc93314fe0590d3afe82191ce571f023c6d